va/va_dec_hevc.h

```diff
@@ -327,7 +475,75 @@ typedef struct  _VASliceParameterBufferHEVC
     uint32_t                va_reserved[VA_PADDING_LOW];
 } VASliceParameterBufferHEVC;
 
+/**
+ * \brief HEVC Extented Slice Parameter Buffer Structure For Long Format
+ *
+ * This data structure contains both baseline HEVC profiles (main, main10)
+ * and extension profiles (range extension and screen content).
+ *
+ * VASliceParameterBufferHEVCExt structure should be accompanied by a
```

Comment:
is the new structure VASliceParameterBufferHEVCExt or VASliceParameterBufferHEVCRext?